implementations/rust/ockam/ockam_core/src/cbor_utils/cow_str.rs

```diff
@@ -77,3 +78,13 @@ impl<'a, S: ?Sized + AsRef<str>> PartialEq<S> for CowStr<'a> {
         self.0 == other.as_ref()
     }
 }
+
+impl CowStr<'_> {
+    /// Join a slice of CowStr into a String.
+    pub fn slice_to_string(data: &[CowStr<'_>]) -> String {
```

Comment:
I have to implement AsRef for CowStr. Would this be ok?

```rust
impl<'a> AsRef<str> for CowStr<'a> {
    #[inline]
    fn as_ref(&self) -> &str {
        self.0.as_ref()
    }
}
```

I added the `inline` attribute because it looks like std implementations of this trait use it.

Reply:
I would not bother with inline unless profiling shows it would benefit the code.